Numsense/Numeral.fs

```diff
@@ -9,3 +9,6 @@ let tryParseEnglish = English.tryParseEnglishImp
 
 let toPolish =        Polish.toPolishImp
 let tryParsePolish =  Polish.tryParsePolishImp
+
+let toRussian =       Russian.toRussianImp Russian.Masculine
```

Comment:
> When simply saying the number, it's always masculine.

That seems like a reasonable motivation for keeping the name simply `toRussian`.

For now, I'd like to keep the API simple, so let's worry about the other cases if it ever becomes relevant.